tests/qos/test_buffer.py

```diff
@@ -967,12 +976,18 @@ def test_headroom_override(duthosts, rand_one_dut_hostname, conn_graph_facts, po
 
     original_speed = duthost.shell('redis-cli -n 4 hget "PORT|{}" speed'.format(port_to_test))['stdout']
     original_cable_len = duthost.shell('redis-cli -n 4 hget "CABLE_LENGTH|AZURE" {}'.format(port_to_test))['stdout']
-    original_profile = duthost.shell('redis-cli hget "BUFFER_PG_TABLE:{}:3-4" profile'.format(port_to_test))['stdout'][1:-1]
-    original_pg_size = duthost.shell('redis-cli hget "{}" size'.format(original_profile))['stdout']
+    if check_qos_db_fv_reference_with_table(duthost) == True:
+        original_profile = duthost.shell('redis-cli hget "BUFFER_PG_TABLE:{}:3-4" profile'.format(port_to_test))['stdout'][1:-1]
+        original_profile_key = original_profile
```

Comment:
Looks like the `original_profile_key` is redundant. We can remove it, make `original_profile` prefixed with the table name, like `BUFFER_PROFILE_TABLE:pg_lossless_100000_5m_profile` and then replace all `original_profile_key` with `original_profile`.
After that, we can remove the change in L1017~L1074 because `original_profile` always contains the table name.

Reply:
Done